src/web/docusaurus/docs/api-services/auth.md

```diff
@@ -1,3 +1,86 @@
+---
+sidebar_position: 5
+---
+
 # Authentication and Authorization
 
-Documentation in construction...
+## Summary
+
+SSO service is required for authentication. Telescope doesn't store user information itself, we just send it to Seneca's SSO. From that interaction, Telescope creates a token that will only work for a limited amount of time. The token is used in authenticating in various Telescope services, without a need to make requests to Seneca's servers every time. The same token system is also used throughout our Supabase back end.
```

Comment:
Maybe we can reword it from:
`The token is used in authenticating in various Telescope services, without a need to make requests to Seneca's servers every time` 
 to -> `The token is used to authenticate various Telescope services without sending requests to Seneca's server every time.`

Change `back end`  -> `backend`